server/server.go

```diff
@@ -605,6 +607,26 @@ func (s *Server) sendError(reply string, err error) {
 	s.publish(reply, proxy.NewErrorResponse(err))
 }
 
+func (s *Server) logEventWithTimestamp(e ari.Event, ts *int64) {
+	eventTs := time.Now().Unix()
+	if *ts == 0 {
+		s.Log.Debug("event received", "kind", e.GetType(), "time received", eventTs)
```

Comment:
missing the "keys" and "key" fields from above in these log lines - we need those.

Reply:
ty... going too fast